chef/automate
# Desktop deployment

### :nut_and_bolt: Description: What code changed, and why?
Introducing a flag to install automate in desktop mode. To install automate with Desktop mode enable run `chef-automate deploy --product desktop`. To enable Desktop mode in the hab studio, run `enable_desktop` in the hab studio. To see that desktop is enabled in the UI, look for the "https://localhost/automate.conf.js" request on any page in Automate. Ensure the response has "desktop" in the products list. 

### :chains: Related Resources

https://github.com/chef/automate/issues/3244

### :athletic_shoe: How to Build and Test the Change
1. `build components/automate-cli && build components/automate-deployment && start_all_services`
1. Run this command for reference below. `curl -f --insecure -H "api-token: ${token}" https://localhost/automate.conf.js`
1. Run `enable_desktop`
1. Run this command to simulate a Desktop install. The below response with "desktop" added should be returned`curl -f --insecure -H "api-token: ${token}" https://localhost/automate.conf.js`
```
parseStaticAutomateConfig({
  "products": [
    "desktop"
  ]
})
```
### :white_check_mark: Checklist
